src/OfxNet/Sgml/SgmlParser.cs

```diff
@@ -32,15 +34,39 @@ public SgmlElement Parse(string path, Encoding encoding)
             var line = reader.ReadLine();
             ++this.lineNumber;
 
-            if (string.IsNullOrWhiteSpace(line) == false)
+            if (!string.IsNullOrEmpty(line))
             {
-                this.ProcessLine(line);
+                var tags = line.Split("<").ToList();
+                tags.ForEach(t =>
```

Comment:
My preference is not to introduce a dependency on LINQ
```suggestion
                foreach (string tag in tags)
```